crypto/evp/evp_local.h

```diff
@@ -370,7 +356,12 @@ OSSL_PARAM *evp_pkey_to_param(EVP_PKEY *pkey, size_t *sz);
             *arglen = pksize;                                     \
             return 1;                                             \
         }                                                         \
-        if (*arglen < pksize && !SSL_MASTER_KEY(ctx, *arglen)) {  \
+        int pad_mode = 0;                                         \
```

Comment:
coding style nit: we don't intermix declarations and code.  I.e. this needs to move up ten lines.

Reply:
Fair point. Fixed it, and also a missing '*' in arglen